### DISCUSSION TITLE
Relative timestamps are incorrect

### DISCUSSION THREAD

**Discussion by InSyncWithFoo** [community/community#114832] [Repositories]
### Select Topic Area

Bug

### Body

This screenshot was taken just now, on March 30, 2024 (from [here](https://github.com/InSyncWithFoo/pyright-for-pycharm/blame/master/README.md)):

![4 days ago - Feb 25, 2024](https://github.com/community/community/assets/122007197/18b818f8-baed-4288-829a-7b9496f93bc1)

Apparently GitHub is telling me my clock is one month later than it should be.

Is this a common problem or there's something wrong on my part?

**Comment by rly** [comment]
I came across what looks to be the same bug just now. I just took this screenshot from the numpy commits log: https://github.com/numpy/numpy/commits/main/?since=2024-01-31&until=2024-02-29
![image](https://github.com/community/community/assets/310197/06f6d566-4d8c-44e2-aa2d-10841e52e200)

The commits above the header "Commits on Feb. 28, 2024" are from February 29. All commits on Feb. 28 are showing as "now". All commits on Feb. 27 are showing as "yesterday" or "2 days ago". It looks like the "yesterday" ones are commits after 12 PM PST and "two days ago" are commits before 12 PM PST. Similar for the earlier days. Commits on Jan. 30 say "one month ago". Commits on Jan. 29 say "two months ago". All commits between Jan. 30 and Feb. 28 appear to be relative to Feb. 28. Leap year bug?

**Comment by mnordhoff** [comment]
I just noticed this bug on an [issue](https://github.com/google/ngx_brotli/issues/166) reported February 29, which the page says is "now".

**Comment by budwaiser4** [comment]
Same problem here

**Comment by InSyncWithFoo** [comment]
Seems to be fixed now.
